diff --git a/Tweak.xm b/Tweak.xm
index 1234567..abcdefg 100644
--- a/Tweak.xm
+++ b/Tweak.xm
@@ -1,5 +1,12 @@
+#import <Foundation/Foundation.h>
+#import <Foundation/NSPredicate.h>
+#import <Foundation/NSString.h>
+#import <Foundation/NSTimer.h>
+#import <Foundation/NSProcessInfo.h>
+#import <Foundation/NSUserDefaults.h>
+#include <string.h>
 #import <substrate.h>
 #import <sys/sysctl.h>
 #import <version.h>
 #import "Header.h"
